packages/components/src/components/domainproperties/dataclasses/DataClassPropertiesPanel.tsx

```diff
@@ -62,7 +64,8 @@ export class DataClassPropertiesPanel extends React.PureComponent<Props> {
     }
 }
 
-class DataClassPropertiesPanelImpl extends React.Component<Props, State> {
+//Note: exporting this class for jest test case
```

Comment:
Why not use the exported Panel class above?

Reply:
Because the jest tests get mad (at least with the test cases I was trying) with how that exported panel above uses the DomainPropertiesPanelProvider and related DomainPropertiesPanelContext.